src/broker/Makefile.am

```diff
@@ -93,14 +95,16 @@ test_ldadd = \
 	$(top_builddir)/src/common/libflux-internal.la \
 	$(top_builddir)/src/common/libtap/libtap.la \
 	$(ZMQ_LIBS) \
+	$(JANSSON_LIBS) \
 	$(PMIX_LIBS)
 
 test_ldflags = \
 	-no-install
 
 test_cppflags = \
         -I$(top_srcdir)/src/common/libtap \
-        $(AM_CPPFLAGS)
+        $(AM_CPPFLAGS) \
+	$(JANSSON_CFLAGS)
```

Comment:
Inconsistent whitespace here (use tabs I think).

Also `$(JANSSON_CFLAGS)` is also being added to `$(AM_CPPFLAGS)`, which is included here, so this is a dup.